### PR TITLE
[DEV-4919] Support table alias in read_sql_query

### PR DESCRIPTION
**Pull request by gaetano-guerriero**

### CODE REVIEW

File: server/athenian/api/async_utils.py [hunk 1]
```diff
@@ -511,9 +490,32 @@ def _extract_integer_columns(
     }
 
 
+def _unwrap_col(col: Column):
+    if isinstance(col, Label):
+        return col.element
```
Comment (vmarkovtsev) [inline server/athenian/api/async_utils.py:495]:
```suggestion
        return _unwrap_col(col.element)
```

File: server/athenian/api/async_utils.py [hunk 2]
```diff
@@ -511,9 +490,32 @@ def _extract_integer_columns(
...
+        except AttributeError:
+            pass
+
+    return col
+
+
+def _get_col_info(col: Column):
+    """Get the `info` attribute of the column, unwrapping Label and Alias if needed."""
+    col = _unwrap_col(col)
+    return getattr(col, "info", {})
```
Comment (vmarkovtsev) [inline server/athenian/api/async_utils.py:508]:
```suggestion
    return getattr(_unwrap_col(col), "info", {})
```

File: server/athenian/api/async_utils.py [hunk 3]
```diff
@@ -511,9 +490,32 @@ def _extract_integer_columns(
...
+
+
+def _get_col_nullable(col: Column):
+    col = _unwrap_col(col)
+    return getattr(col, "nullable", False)
```
Comment (vmarkovtsev) [inline server/athenian/api/async_utils.py:513]:
```suggestion
    return getattr(_unwrap_col(col), "nullable", False)
```

File: server/athenian/api/internal/miners/github/pull_request.py [hunk 4]
```diff
@@ -1119,6 +1119,7 @@ async def fetch_jira():
                     ],
                 )
                 df[PullRequest.node_id.name] = df[PullRequest.node_id.name].astype(int)
+                df[_issue.project_id.name] = df[_issue.project_id.name].astype("S8")
```
Comment (vmarkovtsev) [inline server/athenian/api/internal/miners/github/pull_request.py:1122]: Let me debug this separately, we shouldn't need this, in theory. Can be another pandas glitch.

Comment (gaetano-guerriero) [reply server/athenian/api/internal/miners/github/pull_request.py:1122]: https://github.com/athenianco/athenian-api/runs/8249382270?check_suite_focus=true#step:12:232
This is the failing test without

Comment (vmarkovtsev) [reply server/athenian/api/internal/miners/github/pull_request.py:1122]: OK, I will investigate after the merge.

**Review by vmarkovtsev — APPROVED** [review]

File: server/athenian/api/internal/miners/github/pull_request.py (hunk 4 above)
Comment (vmarkovtsev) [reply server/athenian/api/internal/miners/github/pull_request.py:1122]: I see. This is a new dataframe without dtypes. All right.

**Pull request merged**